subsystems/hexe_edm/data/slowcontrol.json

```diff
@@ -185,16 +221,21 @@
         return function(e, o) {
         if (e !== null) {
           return;}
-        $('.' + x.id).val(x.v);
+        if (o.rows.length != 1) {
+          alert(x.value + " not found");
+          return;
+        }
+        var v = o.rows[0].value;
+        $('.' + x.name).val(v);
         last_sync = new Date();
         };
     };
 
     function sync_vars() {
-        for (var va in vars) {
-            nedm.get_database('hexe_edm').get_most_recent_value(va.v, post_value(va));
+        for (var v in vars) {
+            nedm.get_database('hexe_edm').get_most_recent_value(vars[v].value, post_value(vars[v]));
         }
-        time_out = setTimeout(sync_vars, 5000);
+        time_out = setTimeout(sync_vars, 2000);
```

Comment:
It is clear that you are requiring getting the most recent values of multiple variables... I'll build this in to the API...